spec_mode/mirimrs/notebook1/MRS_Notebook1.ipynb

```diff
@@ -140,14 +140,6 @@
     "- mirisim does not add all of the necessary header keywords for the pipeline to know how to do background subtraction, identify source type, etc.  In order to get these APT-derived keywords correct they will need to be set manually."
```

Comment:
<p>This is an example of a cell with a block comment that could be transitioned into Markdown.</p><p> </p><p>It's subjective, but I would say that higher level comments about the purpose of a cell merit separate descriptions in Markdown cells. Meanwhile, lower level comments that describe specific variables or are only a couple of words are fine as inline comments.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/jwebbinar_prep/pull/45/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>

Reply:
I've tried to be a bit more consistent about this.  The idea is that anything describing what the cell is doing in detail is useful to keep in the cell so that it can be moved around to other notebooks easily by a user, while editorial comments relevant to the logical flow of the notebook itself make sense to be in markdown as those would *not* be copied by a user making a notebook of their own.